backend/src/zcore/config/urls_public.py

```diff
@@ -3,22 +3,24 @@
 from django.contrib import admin
 from django.conf import settings
 from django.conf.urls.static import static
+from decorator_include import decorator_include
 
 from zcore.apps.shared.platformauth.views import (
     PlatformUserLoginView,
 )
+from zcore.core.utils import internal_access_only
 
 urlpatterns = [
     re_path(
         r"^admin/login/$",
-        PlatformUserLoginView.as_view(),
+        decorator_include(internal_access_only, "zcore.apps.shared.platformauth.urls"),
```

Comment:
here we can apply the decorator to direct view, right ?
`internal_access_only(PlatformUserLoginView.as_view())`

Reply:
yeah, I'll add the change